sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/models/application-offering-change-request.dto.ts

```diff
@@ -173,6 +174,14 @@ export class StudentApplicationOfferingChangeRequestAPIInDTO {
   applicationOfferingChangeRequestStatus: ApplicationOfferingChangeRequestStatus;
 }
 
+export class ApplicationOfferingChangeAssessmentAPIInDTO {
+  @IsNotEmpty()
+  @MaxLength(NOTE_MAX_LENGTH)
```

Comment:
If we are getting the max from the DB perspective it should come from the `NOTE_DESCRIPTION_MAX_LENGTH` that represents the DB max length for the table `sims.notes`.